presidio-analyzer/presidio_analyzer/pattern_recognizer.py

```diff
@@ -96,6 +96,19 @@ def validate_result(self, pattern_text):
         """
         return None
 
+    # pylint: disable=unused-argument, no-self-use, assignment-from-none
+    def invalidate_result(self, pattern_text):
```

Comment:
Do we currently have just one usage for this? If yes, maybe we should wait with abstraction